File: Java/Java EE/Servlets/FirstServlet/src/Model/entity/StuffedToys.java
package Model.entity;


import Model.entity.abstracts.Toy;

public class StuffedToys extends Toy {
    private String person;
    private String gender;
    private int ageLimit;

    public StuffedToys() {
    }

    public StuffedToys(double price, double weight, double size, String person, String gender, int ageLimit) {

        super(price, weight, size);
        this.person = person;
        this.gender = gender;
        this.ageLimit = ageLimit;
    }

    public String getPerson() {
        return person;
    }

    public void setPerson(String person) {
        this.person = person;
    }

    public String getGender() {
        return gender;
    }

    public void setGender(String gender) {
        this.gender = gender;
    }

    public int getAgeLimit() {
        return ageLimit;
    }

    public void setAgeLimit(int ageLimit) {
        this.ageLimit = ageLimit;
    }

    @Override
    public String toString() {
        return new StringBuilder().append("\nStuffed Toys " + super.toString() + "\n\tPerson: " + person + "\n\tGender: " + gender + "\n\tAge limit: " + ageLimit).toString();
    }
}

File: Csharp/Lab8/Lab8/Lab8/cng/Model/Entity/StuffedToys.cs
using System.Text;
using Lab8.cng.Model.Entity.Abstracts;

namespace Lab8.cng.Model.Entity
{
    public class StuffedToys : Toy
    {
        private string _person;
        private string _gender;
        private int _ageLimit;

        public StuffedToys() {
        }

        public StuffedToys(double price, double weight, double size, string person, string gender, int ageLimit) : base(price, weight, size){

            
            this._person = person;
            this._gender = gender;
            this._ageLimit = ageLimit;
        }

        public string GetPerson() {
            return _person;
        }

        public void SetPerson(string person) {
            this._person = person;
        }

        public string GetGender() {
            return _gender;
        }

        public void SetGender(string gender) {
            this._gender = gender;
        }

        public int GetAgeLimit() {
            return _ageLimit;
        }

        public void SetAgeLimit(int ageLimit) {
            this._ageLimit = ageLimit;
        }

        
        public override string ToString() {
            return new StringBuilder().Append("\nStuffed Toys " + base.ToString() + "\n\tPerson: " + _person + "\n\tGender: " + _gender + "\n\tAge limit: " + _ageLimit).ToString();
        }
    }
}